sdk/appconfiguration/azure-spring-cloud-appconfiguration-config/src/main/java/com/azure/spring/cloud/config/AppConfigurationPropertySourceLocator.java

```diff
@@ -123,7 +122,7 @@ public AppConfigurationPropertySourceLocator(AppConfigurationProperties properti
         startup.set(false);
 
         // Loading configurations worked. Setting attempts to zero.
-        CalculatedBackoffTime.resetAttempts();
+        StateHolder.clearAttempts();
```

Comment:
Do we also need to clear attempts tracked in `State` here?

Reply:
No, we create a new State to replace the current one on a successful refresh. So, then it will be back to 1.

Reply:
#Resolved